lua/neogit.lua

```diff
@@ -59,14 +59,14 @@ local neogit = {
     if opts ~= nil then
       config.values = vim.tbl_deep_extend("force", config.values, opts)
     end
-    if not config.values.disable_signs then
-      signs.setup()
-    end
+
```

Comment:
This unconditionally sets up signs

The README still mentions `disable_signs`

Reply:
The check was just moved into the `setup()` function: https://github.com/TimUntersberger/neogit/pull/501/files/cad85a236022389a127454f0a69c5d5fd125d6a8#diff-26497fce652f1fc707efdf4c7a8db4f3e6f96c3e7cd4c22c9b07c7515ac237b1R21